clang/lib/DPCT/DPCT.cpp

```diff
@@ -603,6 +601,23 @@ int runDPCT(int argc, const char **argv) {
   }
 
   initWarningIDs();
+
+  DpctInstallPath = getInstallPath(argv[0]);
+
+  if (PathToHelperFunction) {
+    SmallString<512> pathToHelperFunction(DpctInstallPath);
```

Comment:
Install path include "bin",  so please remove the "bin"

Reply:
Bin is not included in the DPCTInstallPath variable. Checked.